.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,39 @@
+# https://docs.github.com/en/actions
+
+name: "Integrate"
+
+on:
+  pull_request: null
+  push:
+    branches:
+      - "master"
+
+jobs:
+  tests:
+    name: "Tests"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v3"
+
+      - name: "Set up PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          extensions: "none, dom, json, mbstring, tokenizer, xml, xmlwriter"
+          php-version: "${{ matrix.php-version }}"
+
+      - name: "Set up problem matchers for PHP"
+        run: "echo \"::add-matcher::${{ runner.tool_cache }}/php.json\""
+
+      - name: "Run tests"
+        run: "php tests/run-tests.php"
+        env:
+          NO_INTERACTION: "true"
```

Comment:
I think we should add some useful arguments. First `-q` is equivalent to setting `NO_INTERACTION`. Then, at least in the long run, parallel test execution makes sense (`-j2` or maybe `-j3` seem to be good for GH actions, currently). And `--show-diff` is helpful to diagnose failing tests.

We could also create a `junit.out.xml` by setting the environment variable `TEST_PHP_JUNIT` to the desired filename, and then packing the file for download. Might be overkill for now, though.